src/Views/batchprocessor.html

```diff
@@ -105,25 +108,27 @@ <h3>Select Flow Statistics</h3>
                     <span ng-if="!vm.flowStatsAllChecked">Unselect All Flow Statistics</span>
                 </button>
                 <!-- create list of flow stats checkboxes -->
-                <table class="table table-condensed">
-                    <thead class="param-table">
-                        <tr>
-                            <th >Select</th>
-                            <th class="col-xs-7">Statistic Group</th>
-                            <th class="col-xs-3"></th>
-                        </tr>
-                    </thead>
-                <tbody ng-repeat="flowStat in vm.flowStatsList">
-                    <tr>
-                        <td>
-                            <checkbox large ng-model="flowStat.checked" ng-change="vm.setRegionStats(flowStat)">
-                            </checkbox>
-                        </td>
-                        <td><span>&nbsp;&nbsp;{{flowStat.statisticGroupName}}</span></td>
-                    </tr>
-                </tbody>
-
-                </table>
+                <div ng-hide="!vm.flowStatsListSpinner"><i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i></div>

```

Comment:
Can you please add a little space between the the blue button and the spinner?

Reply:
Done!